modules/end-user-guide/pages/enabling-go-artifact-repositories.adoc

```diff
@@ -0,0 +1,55 @@
+:navtitle: Go
+:keywords: go, artifact-repository, artifact-repositories
+:page-aliases: .:using-go-artifact-repositories
+
+[id="enabling-go-artifact-repositories"]
+= Enabling Go artifact repositories
+
+You can enable a Go artifact repository in Go workspaces that run in a restricted environment.
+
+.Prerequisites
+
+* You are not running any Go workspace.
+
+include::partial$snip_warning-about-applying-a-configmap-that-sets-environment-variables.adoc[]
+
+.Procedure
+
+. Apply the Secret for the TLS certificate:
```

Comment:
Secret or secret? I'm not sure the capital S is necessary here (same for the other occurrences below).